QUICKSTART.md

```diff
@@ -0,0 +1,279 @@
+# **KubeStellar** Quickstart
+
+## Required Packages:
+   - [docker](https://docs.docker.com/get-docker/)
```

Comment:
We should support free alternatives to Docker.

Reply:
Fixed. Update with link with reference to installation of Docker Engine.

Reply:
Docker is not free on Mac to users in medium and large businesses.